README.md

```diff
@@ -44,6 +44,21 @@ following actions on the project and its submodules:
 
 > **_NOTE:_** The minimum supported Gradle version to use this plugin is v6.9
 
+<details>
+<summary> 🚚 Migrating from v4 </summary>
+
+### Breaking Changes 🚨
```

Comment:
Let's remove this title. and add the following phrase instead:

Version 5 of the Gradle Artifactory Plugin includes the following breaking changes compared to version 4.